smoke/ip6_forward_frr_test.sh

```
@@ -34,15 +31,15 @@ set_ip_route fd00:f00:2::/64 $p2
-ip netns exec n-$p2 ping6 -i0.01 -c3 -n fe80::d2f0:cff:feba:a412
-ip netns exec n-$p1 ping6 -i0.01 -c3 -n fd00:f00:2::2
-ip netns exec n-$p2 ping6 -i0.01 -c3 -n fd00:f00:1::2
-ip netns exec n-$p1 ping6 -i0.01 -c3 -n fd00:ba4:2::2
-ip netns exec n-$p2 ping6 -i0.01 -c3 -n fd00:ba4:1::2
-ip netns exec n-$p1 ping6 -i0.01 -c3 -n fd00:ba4:1::1
-ip netns exec n-$p2 ping6 -i0.01 -c3 -n fd00:ba4:2::1
-ip netns exec n-$p1 traceroute -N1 -n fd00:ba4:2::2
-ip netns exec n-$p2 traceroute -N1 -n fd00:ba4:1::2
-ip netns exec n-$p1 traceroute -N1 -n fd00:f00:2::2
-ip netns exec n-$p2 traceroute -N1 -n fd00:f00:1::2
+ip netns exec $p1 ping6 -i0.01 -c3 -n $(llocal_addr $p1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)